nepali/about_nepal.md

```diff
@@ -2,4 +2,12 @@
 र पुरातातिवक समपदाहरूको बखान अनतराषिटय समुदायमा 
 पिन ठुलो छ । 
 
-थप जानकारीको लािग [यहाँ िथचनुहोस् ।](http://en.wikipedia.org/wiki/Nepal)
\ No newline at end of file
+थप जानकारीको लािग [यहाँ िथचनुहोस् ।](http://en.wikipedia.org/wiki/Nepal)
+
+
+
+नेपालको भूगोल
+
+नेपाल एउटा सानो, भूपरिवेष्ठित, पहाडै पाहाड भएको, दुई ठूला राष्ट्र भारत र चीनको बीच अवस्थित बहुजातीय, बहुभाषिक, बहुधार्मिक, बहुसांस्कृतिक विविधतापूर्ण भौगोलिक बिशेसता युक्त स्बतन्त्र, अटल एवं अखण्ड राष्ट्र हो । भौगोलिक रुपमा नेपाल २६० २२” उत्तर देखी ३०० २७” उत्तर अक्षाशं र ८०० ४” पूर्व देखी ८८० १२” पूर्वी देशान्तरभित्र दक्षिण एशियाको भारतीय उपमहाधौप एवं सार्क क्षेत्र (दक्षिण एसियाली क्षेत्रीय सहयोग संगठन) भित्र अवस्थित र मकर रेखाभन्दा उत्तरतिरको समशीतोष्ण भूमागसम्म पूर्व पश्चिम करीब (सरदर) ८८५ कि मी र उत्तर दक्षिण १९३ कि मी क्षेत्रमा करीब १,४७,१८१ वर्ग कि मी मा फैलिएको छ र नेपालले पृथ्वीको कुल भूमिको ०.०३ प्रतिशत र एशियाको कुल भूमिको ०.३ प्रतिशत भाग ओगटेको छ । भौगोलिक अवस्थाको आधारमा नेपाललाई सामान्यता तीन भाग हिमाल, पाहाड, तराई क्षेत्र (समतल भूमि मा बाँड्न सकिन्छ । धरातलीय रुपमा यस राष्ट्रमा समुद्र सतहबाट करीब ७० मीटरमा अवस्थित कचनाकलन, झापा (सबैभन्दा होचो भाग) र संसारको सर्वोच्च स्थल समुद्र सतहबाट ८८४८ मिटरको उचाइमा रहेको सगरमाथा छ ।
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.